lib/Gateway.js

```diff
@@ -662,6 +662,23 @@ function setBinaryPayloadFromSensor (cfg, valueId, offStateValue = 0) {
   }
   return cfg
 }
+
+/**
+ *
+ * @param {import('../types').HassDevice} cfg Hass discovery Configuration
+ * @param {string} devClass Choosen device class
+ * @param {boolean} reversePayload reverse payload order
+ */
+function setBinarySensorPayload (devClass, reversePayload = false) {
```

Comment:
```suggestion

/**
 * Create a binary sensor configuration with a specific device class
 *
 * @param {string} devClass Choosen device class
 * @param {boolean} reversePayload reverse payload order
 * @returns {import('../types').HassDevice} cfg Hass discovery Configuration for the binary sensor
 */
function getBinarySensorConfig (devClass, reversePayload = false) {
```

Change all the other references according to this